src/platform/intel/cavs/platform.c

```diff
@@ -27,6 +27,7 @@
 #include <sof/lib/notifier.h>
 #include <sof/schedule/edf_schedule.h>
 #include <sof/schedule/ll_schedule.h>
+#include <sof/lib/pm_runtime.h>
```

Comment:
Should go after `#include <sof/lib/notifier.h>`.

Reply:
Done